Graphs/NodeNeighbors.js

```diff
@@ -1,29 +1,28 @@
 // https://en.wikipedia.org/wiki/Neighbourhood_(graph_theory)
 
-class Graph{
+class Graph {
   // Generic graph: the algorithm works regardless of direction or weight
-  constructor(){
+  constructor () {
     this.edges = []
   }
 
-  addEdge(node1, node2){
+  addEdge (node1, node2) {
     // Adding edges to the graph
     this.edges.push({
       node1,
       node2
     })
   }
 
-  nodeNeighbors(node){
+  nodeNeighbors (node) {
     // Returns an array with all of the node neighbors
-    let neighbors = []
+    const neighbors = []
```

Comment:
```Suggestion
    const neighbors = new Set()
```
Using a `set` would reduce the time complexity from O(n ^ 2) to O(n). Use the for-of loop preferably to iterate over the edges.

Reply:
Hello! 
This is my first commit to an open source project here on github
Why are sets faster in this case?
I was using lists instead of sets because of [this](https://stackoverflow.com/questions/2831212/python-sets-vs-lists), since I am iterating over every node, but the answer I read is from 10 years ago, so not really the best source.
Thanks for the feedback, really appreciate it:)

Reply:
You are not iterating through it. You are checking if the neighbor is present in the list